packages/design-system/src/TextInput/index.tsx

```diff
@@ -222,6 +222,7 @@ export const InputWrapper = styled.div<{
   isValid?: boolean;
   disabled?: boolean;
   $isLoading?: boolean;
+  readOnly?: boolean;
```

Comment:
If you're changing the API, where is the associated main repo PR? 

read-only text input is what UneditableTextField claims to be. Can you find out what the differences are between them and if they're the same replace all instances of UneditableTextField with them?

Reply:
I'm not changing the API here, I'm just passing this variable to the wrapper in order to fix the style. The prop was already there for the component

Reply:
Ah my bad. Approving.